### PR TITLE
PR acceptance criteria template

### PR DESCRIPTION
**Pull request by amitkrout**
[skip ci]

**What type of PR is this?**
/kind documentation

**What does does this PR do / why we need it**:
New field added for pr acceptance criteria

**Which issue(s) this PR fixes**:

Fixes NA

**How to test changes / Special notes to the reviewer**:
👀

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -22,4 +22,12 @@
 
 Fixes #?
 
+**PR acceptance criteria**:
+
+Unit test (Yes/No):
```
Comment (girishramnani) [inline .github/PULL_REQUEST_TEMPLATE.md:27]: Use a check Box instead

Comment (amitkrout) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done

**Comment by prietyc123** [comment]
Requested changes are addressed. Looks good to me.

/lgtm

**Pull request merged**